examples/article.html

```diff
@@ -0,0 +1,93 @@
+						</li>
+
+					</ul>
+				</section>
+				<section id="facebook" class="modal-dialog modal-content overlay-def mfp-hide">
+					<header class="modal-header">
+						<h2 class="modal-title" id="lbx-title">Facebook</h2>
+					</header>
+					<div class="modal-body">
+						<ul class="list-unstyled lst-spcd">
+							<li>
+								<a href="#" rel="external">[First Facebook account title]</a>
+							</li>
+							<li>
+								<a href="#" rel="external">[Second Facebook account title]</a>
+							</li>
+						</ul>
+					</div>
+					<div class="modal-footer">
+						<button type="button" class="btn btn-sm btn-primary pull-left popup-modal-dismiss">Close<span class="wb-inv">Close overlay</span></button>
+					</div>
+				</section>
+			</div>
+		</div>
+	</div>
+	<div class="row">
+		<div class="col-sm-8">
+			<h2 class="mt-0">Lorem ipsum dolor sit amet</h2>
+			<h3 class="mt-0">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam viverra venenatis feugiat</h3>
+
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam viverra venenatis feugiat. Suspendisse eros est, consequat et sagittis ac, posuere aliquet mauris. Nunc eget quam sem. Suspendisse potenti. Nulla ornare erat ac tincidunt dapibus. Suspendisse at nunc eu lectus mollis viverra. Praesent efficitur massa ac lectus posuere, vitae pellentesque erat condimentum.</p>
+			<p>Interdum et malesuada fames ac ante ipsum primis in faucibus. Praesent tincidunt tortor fringilla, pretium lectus suscipit, placerat ex. Proin facilisis sapien erat, a molestie lorem consequat nec. Nulla sollicitudin pretium mi vitae commodo.</p>
+			<p>Nam at ex ac metus hendrerit condimentum eget et dolor. Quisque molestie libero et est pellentesque, vitae aliquet sapien posuere. Vestibulum at posuere risus, sit amet elementum nulla.</p>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam viverra venenatis feugiat. Suspendisse eros est, consequat et sagittis ac.</p>
+			<p>Interdum et malesuada fames ac ante ipsum primis in faucibus. Praesent tincidunt tortor fringilla, pretium lectus suscipit, placerat ex. Proin facilisis sapien erat, a molestie lorem consequat nec.</p>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam viverra venenatis feugiat. Suspendisse eros est, consequat et sagittis ac.</p>
+			<p>Interdum et malesuada fames ac ante ipsum primis in faucibus. Praesent tincidunt tortor fringilla, pretium lectus suscipit, placerat ex.</p>
+		</div>
+		<div class="col-sm-4">			
+			<h2 class="h6">Download:</h2>
+			<a class="btn btn-default btn-sm btn-block stretched-link" href="#">
+				<span class="glyphicon glyphicon-download-alt mr-2"></span>
+				<span>Inuktitut Version</span>
+				<span class="gc-dwnld-info">(<abbr title="Portable Document Format">PDF</abbr>, 273 <abbr title="KiloByte">KB</abbr>)</span>
+			</a>
+			<h2 class="h6 mb-3">Related report:</h2>
+			<article class="oag-card bg-light">
```

Comment:
Let's keep the style for this card the same as the ones on the petitions catalogue sidebar. So, let's add the class "h5". Also, either add the class "oag-brdr" to this one or remove it from the ones in the petitions catalogue.

Reply:
Please change "mb-1" to "h5" on the H3.